contrib/automation_tests/fragments/top_down_tab.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+import logging
+
+from core.orbit_e2e import E2ETestCase, find_control
+
+
+class VerifyHelloGgpTopDownContents(E2ETestCase):
+    """
+        TODO: This is a more or less plain copy of the old E2E test. If may need to
+        be updated
+        """
```

Comment:
Specify how it should be updated, otherwise if it's unclear the TODO should be removed.

Reply:
Now that you mention it, after I realized that the test wasn't working for the new UI, I already updated it slightly. I've changed the comment, it could still be nice to change the test to not contain e.g. the hard-coded number of cells, there's some tooling for this now in common_controls that could be used.